astropy/timeseries/binned.py

```diff
@@ -119,6 +119,13 @@ def __init__(self, data=None, *, time_bin_start=None, time_bin_end=None,
         if isinstance(time_bin_size, TimeDelta):
             time_bin_size = time_bin_size.sec * u.s
 
+        # If any one of the parameters 'time_bin_start' or 'time_bin_end' is not scalar,
```

Comment:
The [PEP8 conventions](https://www.python.org/dev/peps/pep-0008/#:~:text=Limit%20all%20lines%20to%20a,be%20limited%20to%2072%20characters) mention that you should limit lines to 79 characters and docstrings/comments to 72.

```suggestion
        # If any one of the parameters 'time_bin_start' or 'time_bin_end'
        # is not scalar, an exception is raised.
```